MC/config/common/pythia8/utils/mkpy8cfg.py

```diff
@@ -12,13 +12,19 @@
                    help='The random seed')
 
 parser.add_argument('--idA', type=int, default='2212',
-                   help='PDG code of projectile beam')
+                   help='PDG code of projectile beam A')
 
 parser.add_argument('--idB', type=int, default='2212',
-                   help='PDG code of target beam')
+                   help='PDG code of target beam B')
 
-parser.add_argument('--eCM', type=float, default='13000.',
-                    help='Centre-of-mass energy')
+parser.add_argument('--eBeamA', type=float, default='6499.',
```

Comment:
I suggest you keep the same name as the one used by Pythia, namely `--eA` and `--eB`